testdata/uuid-map-one.csv

```diff
@@ -411,14 +411,14 @@ f0b4ce1f-1224-487e-a993-eef896328dbd,pts_1326266236178
 65aa2a12-148e-4afb-a7d0-8c2e02cec763,pts_132633077501
 c21c7f16-654d-4cba-86f7-d10c946dcc59,pts_132638063383
 83c3c192-659b-4491-9df9-401251781f81,pts_1326526280405
-83c3c192-659b-4491-9df9-401251781f81,pts_1326531280406
 83c3c192-659b-4491-9df9-401251781f81,pts_1326529280404
 83c3c192-659b-4491-9df9-401251781f81,pts_132652862804
+83c3c192-659b-4491-9df9-401251781f81,pts_1326531280406
 790c2c7d-1486-47e8-af0e-168220570f09,pts_132659286728
-790c2c7d-1486-47e8-af0e-168220570f09,pts_132659686728
 790c2c7d-1486-47e8-af0e-168220570f09,pts_1326600403145
 790c2c7d-1486-47e8-af0e-168220570f09,pts_1326593403145
 790c2c7d-1486-47e8-af0e-168220570f09,pts_1326597403145
+790c2c7d-1486-47e8-af0e-168220570f09,pts_132659686728
```

Comment:
How come changing the date format would actually cause stuff to be added here?

Reply:
It just changed the ordering for some reason. This moved from line 418 to 421.